### DISCUSSION TITLE
Google Coral TPU

### DISCUSSION THREAD

**Discussion by pewnflap** [blakeblackshear/frigate#2290] [General]
Very long shot but does anyone happen to have a spare USB or m.2 laying around that i could purchase? I don't mind paying a bit above retail. I'd really like some decreased detection times.

Thanks

**Comment by Baael** [comment]
I've solved shortage issue by buying dev board (mini). There was a little struggle with installing docker, but currently I have one instance of frigate deployed on it, and it is connected to remote HA. 

Another option is mac mini m1. Expensive, but gives a lot of other possibilities, I connected there also sound classification and face recognition.

**Reply by mpfl** [reply]
Tell me more about this face recognition...

**Reply by Baael** [reply]
deepstack and some custom experiments, and for sound - yamnet on audio taken from rtsp streams, works as electronic nanny (detect crying), and enables only when children are in beds (frigate zones)

**Comment by pewnflap** [comment]
> I've solved shortage issue by buying dev board (mini). There was a little struggle with installing docker, but currently I have one instance of frigate deployed on it, and it is connected to remote HA.
> 
> Another option is mac mini m1. Expensive, but gives a lot of other possibilities, I connected there also sound classification and face recognition.

I was considering that as a backup plan. I may have to look into it if i can't find anything else!
